src/common/PersonTile/index.js

```diff
@@ -1,12 +1,25 @@
 import { Name, Picture, Role, Wrapper } from "./styled";
 import { PersonImageAPI } from "../../codesAPI";
+import NoImagePerson from "../../images/noimgperson.svg";
 
-const PersonTile = ({ picture, name, role }) => (
-  <Wrapper>
-    <Picture src={`${PersonImageAPI}${picture}`} />
-    <Name>{name}</Name>
-    <Role>{role}</Role>
-  </Wrapper>
-);
+const PersonTile = ({ picture, name, role }) => {
+  if (!picture) {
+    return (
+      <Wrapper>
+        <Picture src={NoImagePerson} />
+        <Name>{name}</Name>
+        <Role>{role}</Role>
+      </Wrapper>
+    );
```

Comment:
I think this can be simplified a bit, instead of rewriting the entire Wrapper 😎

Reply:
Yes, I saw that you did it in a simpler way, so I think we'll use your solution ;-)